iOS/Scripts/Scripty.xcodeproj/Files_Info.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
(pretty much anything under `Scripty.xcodeproj` is automatically generated by SPM)